tests/integration/test_training_pipeline.py

```diff
@@ -0,0 +1,74 @@
+import os
+os.environ["CUDA_VISIBLE_DEVICES"] = "-1"  # force tests to run on CPU
+
+import json
+from pathlib import Path
+import pytest
+import sys
+from unittest.mock import patch
+import warnings
+
+import graph2tac.tfgnn.train
+
+REL_ERROR_TOLERANCE = 1e-3
```

Comment:
This might be two strong.  Lasse can you try changing this to 1e-2?

Reply:
Or if that doesn't work, `5e-2`.

Reply:
With 5e-2:
```
E       AssertionError: Training statistics don't match expected results. To overwrite expected results, rerun this test with `--overwrite`.
E       assert {'arguments_s...7376709], ...} == {'arguments_s....1e-01]), ...}
E         Omitting 19 identical items, use -vv to show
E         Differing items:
E         {'val_local_arguments_logits_accuracy': [0.37662336230278015, 0.4675324559211731]} != {'val_local_arguments_logits_accuracy': approx([0.37662336230278015 ± 1.9e-02, 0.41558441519737244 ± 2.1e-02])}
E         Use -v to get more diff

tests/integration/test_training_pipeline.py:72: AssertionError
```